packages/universal-components/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@kiwicom/universal-components",
+  "version": "0.0.14",
+  "description": "Component Library for React Native, Expo and React applications",
+  "homepage": "https://github.com/kiwicom/universal-components",
+  "main": "lib/web/index.js",
```

Comment:
Works and looks perfect! Thanks a lot.

The only thing I'm not sure about is that we are using `universal-components` from `/lib`.
So build needs to be created first. We should probably make more clear that calling `yarn workspace @kiwicom/universal-components build` is necessary.

Or better, will it be possible (and will it make sense to you) to use components directly from `universal-components/src`? I think that now it can be confusing that for `@kiwicom/universal-components` you need to create build first, but it is not necessary for `@kiwicom/margarita-components`.

Reply:
You're right, thanks for pointing it out.

I'm still not decided which way is best.

I guess for now I could add a postinstall hook that will build the component library for you (which also means that if you are making changes in universal, you need to rebuild it to use it elsewhere...)

Will need to discuss with @mrtnzlml about monorepo architecture and package usage/release.

Reply:
Thanks. I think that postinstall will work for now.